### PR TITLE
Create a Jenkinsfile to run scan on OpenSearch-projects repos

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description

Since #1181 is not working because of GHA. We will now use Jenkins to automate WhiteSource scan. All repos included in `wss-scan.config` are tested to be running properly in Jenkins. "OpenSearch" is not included since we have encountered some issues scanning it within docker container. 

Refer to #1157, WhiteSource integration may not work correctly at this time. They have an issue on their side. We are switching to our Plan B using Jenkins for automating WhiteSource scan based on old script from ODFE. This workflow will update the scan results in main product "OpenSearch" on WhiteSource dashboard.

### Issues Resolved
part of #1157 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dblock — APPROVED** [review]

**Comment by dblock** [comment]
What do you think about moving the list of repos to scan into Jenkinsfile and use the list to generate stages that are then executed in parallel?

**Comment by zelinh** [comment]
> What do you think about moving the list of repos to scan into Jenkinsfile and use the list to generate stages that are then executed in parallel? (e.g. kicks off a job with the argument of repo to scan, then waits for all the started jobs, or maybe there's a way to create stages dynamically).

The reason I keep the list of repos in `wss-scan.config` is that I want to make minimum change to what we had in our repo. But I do like the idea that we have this list in Jenkinsfile as stages; passing the repo as argument or just use the default list seems to be a good idea for other people to kickoff scan. Maybe we could create an issue for this feature as enhancement?

File: tools/vulnerability-scan/wss-scan.sh [hunk 1]
```diff
@@ -31,14 +31,16 @@ fi
 if [ ! -f "wss-unified-agent.jar" ]
 then
   # Download the WhiteSource Agent 
-  wget -q https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
-  # The version 20.9.2.1 has been tested and can be used if a specific version is required
-  #wget -q  https://github.com/whitesource/unified-agent-distribution/releases/download/v20.9.2.1/wss-unified-agent.jar
+  curl https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar --output wss-unified-agent.jar
+  # The version 21.11.1 has been tested and can be used if a specific version is required
```
Comment (peternied) [inline tools/vulnerability-scan/wss-scan.sh:35]: Can we lock to a version?  I would rather we intentional rev the version and see something break then get surprised when we start downloading a new major release

Comment (zelinh) [reply tools/vulnerability-scan/wss-scan.sh]: Changed. We will now stick with unified agent v21.11.1 until further change.

**Review by peternied — APPROVED** [review]

**Pull request merged**